src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.scss

```diff
@@ -80,6 +81,11 @@
   font-weight: $normal;
 }
 
+.externalRefLink {
+  @extend .normalText;
+  font-size: 12px;
+}
```

Comment:
Notice how prominent external reference labels are?
![image](https://user-images.githubusercontent.com/6834224/135548451-d2393d5b-409c-4f5b-92c7-921ea3fcfefa.png)

According to the [XD](https://xd.adobe.com/view/97a926d4-0e6a-4e16-83ad-75b77a8cba48-c2fa/screen/63a15b5d-b168-402b-9aa6-0ef44a13907d?fullscreen), this doesn't look right.

Reply:
✅